### PR TITLE
feat: Initial commit

### PR DESCRIPTION
**Pull request by surajsbharadwaj**
### Description

PowerVS instance creation with storage configuration.

#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [x] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version.

Your notes help the merger write the commit message for the PR that is published in the release notes for the module.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [x] If relevant, a test for the change is included or updated with this PR.
- [x] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

**Comment by surajsbharadwaj** [comment]
@ocofaigh please trigger the pipeline :)

**Review by kbiegert — APPROVED** [review]

File: cra-tf-validate-ignore-goals.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            "is_valid": true
+        },
+        {
+            "scc_goal_id": "3000408",
+            "description": "Check whether Flow Logs for VPC are enabled",
+            "ignore_reason": "Flow logs are enabled, so seems to be CRA bug. Tracking in https://github.ibm.com/oneibmcloud/CD-CRA/issues/1779",
+            "is_valid": false
+        }
+    ]
+}
```
Comment (ocofaigh) [inline cra-tf-validate-ignore-goals.json:41]: You need to create a new file called `cra-tf-validate-ignore-rules.json` and delete this file. This file is referencing SCC v1 goals which are now replaced by SCC rules.

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@surajsbharadwaj Can you please address https://github.com/terraform-ibm-modules/terraform-ibm-powervs-instance/pull/2/files#r1210048889 before I trigger again

Perhaps just delete the file until we need to enable CRA

File: cra-tf-validate-ignore-goals.json (hunk 1 above)
Comment (surajsbharadwaj) [reply cra-tf-validate-ignore-goals.json:41]: can you point to a repo ? because the landing zone is also using the same file

Comment (ocofaigh) [reply cra-tf-validate-ignore-goals.json:41]: Here is an example https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/blob/main/cra-tf-validate-ignore-rules.json however you will have different rule IDs that you may need to ignore. You would have to run CRA manually to see if you need to skip any failing rules

Comment (surajsbharadwaj) [reply cra-tf-validate-ignore-goals.json]: Deleted the file

**Comment by surajsbharadwaj** [comment]
@ocofaigh please run it again :)

**Comment by surajsbharadwaj** [comment]
@ocofaigh done.

File: README.md [hunk 2]
```diff
@@ -1,168 +1,127 @@
...
-### Commit your code and submit your module for review
-
-1.  Before you commit any code, review [Contributing to the IBM Cloud Terraform modules project](https://terraform-ibm-modules.github.io/documentation/#/contribute-module) in the project documentation.
-1.  Create a pull request for review.
-
-### Post-merge steps
-
-After the first PR for your module is merged, follow these post-merge steps:
+[![Graduated (Supported)](https://img.shields.io/badge/status-Graduated%20(Supported)-brightgreen?style=plastic)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+[![build status](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/actions/workflows/ci.yml/badge.svg)](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/actions/workflows/ci.yml)
```
Comment (ocofaigh) [inline README.md:6]: `terraform-ibm-landing-zone` reference needs to be updated here (2 places)

File: README.md [hunk 3]
```diff
@@ -1,168 +1,127 @@
...
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-powervs-infrastructure?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/releases/latest)
```
Comment (ocofaigh) [inline README.md:9]: pointing to wrong repo `terraform-ibm-powervs-infrastructure`

**Review by ocofaigh — CHANGES_REQUESTED** [review]
See comments. Also the content in `solutions/default/README.md` needs to be updated

**Comment by surajsbharadwaj** [comment]
@ocofaigh thank you for those comments. All fixed :)

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**